src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -23,7 +23,7 @@ private string NuGetCachePath
             {
                 if (string.IsNullOrEmpty(_nugetCachePath))
                 {
-                    _nugetCachePath = PackageDependencyProvider.ResolvePackagesPath(null, null);
+                    _nugetCachePath = NuGetPathContext.Create(new NullSettings()).UserPackageFolder;
```

Comment:
@emgarten - can you answer that?

I used JustDecompile to look at the code, and it for sure takes `NUGET_PACKAGES` into account.  Not sure on the other locations...  

This call is odd in general.  NuGet wants a .sln or a project.json folder so it can search up directories for NuGet.config files. But here we are just saying "give me SOME package folder".